.github/workflows/artifacts.yaml

```diff
@@ -67,12 +67,13 @@ jobs:
         with:
           images: ${{ steps.image-name.outputs.value }}
           flavor: |
-            latest = false
+            latest = true
           tags: |
             type=ref,event=branch
             type=ref,event=pr,prefix=pr-
             type=semver,pattern={{raw}}
             type=raw,value=latest,enable={{is_default_branch}}
```

Comment:
do we know why doesn't this work?

Reply:
it worked, built an image on tag 0.0.1-dev, also set latest to the new image. my only problem was about the branches. the latest now points to a build made from dev branch